src/po/swappy.desktop.in

```diff
@@ -3,9 +3,11 @@ Name=Swappy
 GenericName=Annotation Tool
 GenericName[fr]=Outil d'annotation
 GenericName[pt_BR]=Ferramenta de Anotação
+GenericName[de]=Anmerkungswerkzeug
```

Comment:
minor do you mind putting it in alphabetical order ? (before the `fr` one)

Reply:
@jtheoof done. see https://github.com/jtheoof/swappy/pull/95/commits/01639cf2f7e7f7ee95963332b29116ad927bb297
for some reason it didn't pass the first check?! 😕

Reply:
Your commit is not respecting the guidelines in the README. I suggest to squash both commits and use: `fix(i18n): add german translations to desktop file`

Reply:
Never mind, I could do it from the Github UI. Thanks for the update.